pages/index.html

```diff
@@ -4,9 +4,31 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Karina Gorkava portfolio</title>
+  <link href="../stylesheets/style.css" rel="stylesheet" />
 </head>
 <body>
-  
+  <header>
+    <nav>
+      <ul class="nav-item">
+        <li class="scaler"><a href="index.html" target = "_self">Home</a></li>
+        <li class="scaler"><a href="about.html" target = "_self">About me</a></li>
+        <li class="scaler"><a href="portfolio.html" target = "_self">Projects</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main class="intro-container">
+    <section class="intro-content">
+        <h1 class="heading-large">Hi, I'm Karina.</h1>
+        <p class="paragraph-large">I am a software development student (Cohort 16, Expected to graduate in July 2022) at Ada Developers Academy in Seattle.<br>Our curriculum covers Python, HTML/CSS, Flask and Javascript.</p>
+        <button type="button" class="cyber" onclick="location.href='portfolio.html'">
+          <img src="../images/button.svg" alt="button"/>
+          <div class="cyber-text">SEE MY WORK ></div>
```

Comment:
We don't put `div` tags inside `button` tags. 

```suggestion
          <span class="cyber-text">SEE MY WORK ></span>
```